cdap-ui/app/cdap/services/ThemeHelper.ts

```diff
@@ -20,7 +20,7 @@ import isColor from 'is-color';
 import isBoolean from 'lodash/isBoolean';
 
 interface IThemeJSON {
-  "spec-version": string;
+  "spec-version"?: string;
```

Comment:
was this a requirement to make it optional? I don't think this should be optional